demos/html5/tinymce5/index.html

```diff
@@ -4,7 +4,7 @@
     <head>
         <title> Demo Tinymce5 </title>
         <meta http-equiv="content-type" content="text/html; charset=UTF-8">
-        <script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/tinymce.min.js" referrerpolicy="origin"></script>
```

Comment:
Do we know if it's better to add the tinymce dependency from the node modules than the original tinymce cloud?

Reply:
I believe so.

Imho it's much appropiate to use the tinymce package as defined on the project package.json file:

For example, this is the current dependency:

```json
  "dependencies": {
    "@wiris/mathtype-tinymce4": "^7.0.0",
    "tinymce": "^4.0.0"
  },
```